components/skin/api/src/main/resources/ApplicationResources_es.properties

```diff
@@ -12,6 +12,8 @@ phenotips.contentTopMenu.moreActionsMenu.history.title=Ver c\u00f3mo el registro
 phenotips.contentTopMenu.moreActionsMenu.exportPDF=Exportar a PDF
 phenotips.contentTopMenu.moreActionsMenu.exportXAR=Exportar como XAR
 phenotips.contentTopMenu.moreActionsMenu.exportJSON=Exportar a JSON
+phenotips.contentTopMenu.familyActionsMenu.deleteAndUnlink=Eliminar la familia y los miembros de desvincular
+phenotips.contentTopMenu.familyActionsMenu.deleteAllMembers=Borrar todos los miembros de la familia y
```

Comment:
This seems wrong as well. `Eliminar` used above and in most other translations, `Borrar` here. How about:

    Eliminar la familia y todos los miembros